src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableDataBlock.tsx

```diff
@@ -2,12 +2,49 @@ import React from 'react';
 import DataBlock, {
   DataBlockProps,
 } from '@common/modules/find-statistics/components/DataBlock';
-import wrapEditableComponent from '@common/modules/find-statistics/util/wrapEditableComponent';
+import Button from '@common/components/Button';
+import classnames from 'classnames';
+import ModalConfirm from '@common/components/ModalConfirm';
+import styles from './EditableDataBlock.module.scss';
 
-type Props = DataBlockProps;
+type Props = {
+  canDelete?: boolean;
```

Comment:
`canDelete?` isn't use.. 
Perhaps we can imply the we can delete, if an `onDelete` was defined